### PR TITLE
Feature/conditional outcome text

### PR DESCRIPTION
**Pull request by rryan0088**
Set up of basic component to allow for rendering dynamic text to the Confirmation screen at the end of the assessment.

Initial design has 5 Optional text areas in Elsa Designer for a user to put in string or javascript values to be rendered on the final page after the assessment has been submitted.  Values can be dynamic (using javascript) or static (using plain text).

Co-authored-by: Jerry Zajac [32240008+jarekzaj@users.noreply.github.com](mailto:32240008+jarekzaj@users.noreply.github.com)
Co-authored-by: Jonny T [90678699+Jonnythan-t@users.noreply.github.com](mailto:90678699+Jonnythan-t@users.noreply.github.com)
Co-authored-by: Ben Hall [3179852+benbhall@users.noreply.github.com](mailto:3179852+benbhall@users.noreply.github.com)
Co-authored-by: Lorna B [93874097+lornacbirchall@users.noreply.github.com](mailto:93874097+lornacbirchall@users.noreply.github.com)
Co-authored-by: Rob Ryan [112186767+rryan0088@users.noreply.github.com](mailto:112186767+rryan0088@users.noreply.github.com)
Co-authored-by: M S Dev [122261875+MSarwarDev@users.noreply.github.com](mailto:122261875+MSarwarDev@users.noreply.github.com)

### CODE REVIEW

File: src/Elsa.CustomActivities.Tests/Activities/ConfirmationScreen/ConfirmationScreenTests.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
+﻿using Elsa.ActivityResults;
+using Elsa.Services.Models;
+using He.PipelineAssessment.Common.Tests;
+using Xunit;
+
+namespace Elsa.CustomActivities.Tests.Activities.ConfirmationScreen
+{
+    public class ConfirmationScreenTests
+    {
+        //[Theory]
```
Comment (jarekzaj) [inline src/Elsa.CustomActivities.Tests/Activities/ConfirmationScreen/ConfirmationScreenTests.cs:10]: Commented out test, so can be removed

File: src/Elsa.Dashboard/src/components/elsa-multiquestion-records-property/elsa-question-screen.tsx [hunk 2]
```diff
@@ -68,6 +68,7 @@ export class ElsaQuestionScreen {
   async componentWillLoad() {
     const propertyModel = this.propertyModel;
     const choicesJson = propertyModel.expressions[SyntaxNames.Json]
+    console.log(choicesJson);
```
Comment (jarekzaj) [inline src/Elsa.Dashboard/src/components/elsa-multiquestion-records-property/elsa-question-screen.tsx:71]: might want to remove this as well

File: src/Elsa.Dashboard/package.json [hunk 3]
```diff
@@ -29,7 +29,6 @@
     "unit-test": "jest"
   },
   "dependencies": {
-    "@elsa-workflows/elsa-workflows-studio": "^2.8.2",
```
Comment (jarekzaj) [inline src/Elsa.Dashboard/package.json:32]: is this intended?

Comment (Jonnythan-t) [reply src/Elsa.Dashboard/package.json:32]: actually not being used because we are not importing anything from elsa directly

**Review by jarekzaj — APPROVED** [review]

**Pull request merged**